cluster-autoscaler/FAQ.md

```diff
@@ -1112,8 +1115,18 @@ The script shall create a directory under the `/tmp`, and logs of the execution-
 Once script is successfully executed, execute following commands to confirm the correctness.
 ```
 # You must see a new commit created by the script containing the commit-hash.
+
 git status 
+```
+<u><b> For syncing with kubernetes/autoscaler >= v1.21.0 </u> </b>
 
+With kuberenets/autoscaler v1.21.0 the `update_vendor.sh` has been updated. We just need to provide the version of k8s dependencies which we want to vendor in , as an argument to the script. For ex, if we want to vendor in `k8s v1.21.0`
+
+```
+ ./update-vendor.sh v1.21.0 
```

Comment:
```suggestion
 ./update-vendor.sh 1.21.0 
```

If I remember correctly then `v` need not be supplied. Could you please check?

Reply:
Thanks will update it.